docs/api/MockProvider.md

```diff
@@ -0,0 +1,73 @@
+---
+title: <MockProvider />
+---
+
+```typescript
+function MockProvider({
+  children,
+  results,
+}: {
+  children: React.ReactChild;
+  results: Fixture[];
+}): JSX.Element;
+```
+
+\<MockProvider /> is a simple substitute provider to prefill the cache with fixtures so the 'happy path'
+can be tested. This is useful for [storybook](../guides/storybook.md) as well as component testing.
+
+## Arguments
+
+### results
+
+```typescript
+interface Fixture {
+  request: ReadShape<Schema, object, any>;
+  params: object;
+  result: any;
+}
+```
+
+This prop specifies the fixtures to use data from. Each item represents a fetch defined by the
+[RequestShape](./RequestShape.md) and params. `Result` contains the JSON response expected from said fetch.
+
+## Returns
+
+```typescript
+JSX.Element
+```
+
+Renders the children prop.
+
+## Example
+
+```typescript
+import { MockProvider } from 'rest-hooks/test';
+
+import ArticleResource from 'resources/ArticleResource';
+import MyComponentToTest from 'components/MyComponentToTest';
+
+const results = [
+  {
+    request: ArticleResource.listRequest(),
+    params: { prop: 10 },
```

Comment:
why is this called prop?

Reply:
haha, i stole it from the issue. I'll make it something more meaningful